src/api/resources/mixins.py

```diff
@@ -260,6 +264,31 @@ def edit(self, request, project_pk, pk):
             return Response(str(err), status=500)
 
 
+class SampleUnitMethodSummaryReport(object):
+    @action(
+        detail=False, methods=["GET"], permission_classes=[ProjectDataAdminPermission]
+    )
+    def xlsx(self, request, project_pk):
+        from ..utils.reports import create_sample_unit_method_summary_report
```

Comment:
Any reason not to put this up top?

Reply:
avoiding circular reference.  Not ideal but I wanted to leverage the existing viewset CSV and since this is a mixin that gets added back to the view set it gets grumpy